submissions/SerhiiYakovenko/html-css-popup/css/style.css

```diff
@@ -0,0 +1,180 @@
+* {
+  margin: 0;
+  padding: 0;
+  color: black;
+}
+
+body {
+  font-family: "Orbitron", sans-serif;
+  background: url(".././img/background.jpg") no-repeat center center fixed;
+  background-size: cover;
+  height: 100%;
+  overflow: hidden;
+}
+
+img {
+  max-width: 100%;
+  max-height: 40px;
+  border: solid 5px transparent;
+}
+
+.release {
+  font-size: 35px;
+  color: yellow;
+  background-color: black;
+  position: fixed;
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+  opacity: 0.6;
+}
+
+.nav {
+  margin: 5px;
+  float: right;
+  display: flex;
+  align-items: center;
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.nav-item {
+  list-style: none;
+  margin: 10px 5px;
+  border: solid 1px transparent;
+}
+
+.nav-item.console {
```

Comment:
At first, you can simplify this selector by removing `.nav-item`. Also, I'd recommend you to simplify styles, because `.nav-item` has already  these styles: `list-style: none;` and `border: solid 1px transparent;`. No need to duplicate them. And it seems you don't need `list-style: none;` for `.console` in general, so just remove all of them and it will work properly

Reply:
True, `.console` was redundant at all. 
Thanks for pointing to it.